airflow/providers/apache/sqoop/hooks/sqoop.py

```diff
@@ -61,14 +61,15 @@ def __init__(
         hcatalog_database: str | None = None,
         hcatalog_table: str | None = None,
         properties: dict[str, Any] | None = None,
+        libjars: str | None = None,
```

Comment:
Maybe this idea to extend to `files` and `archives` too seeing as they are ["not typically used with Sqoop, but they are included as part of Hadoop’s internal argument-parsing system."](https://sqoop.apache.org/docs/1.4.2/SqoopUserGuide.html#_using_generic_and_specific_arguments)?